### PR TITLE
ncm-authconfig: sssd: remove unsupported force_timeout domain option

### PR DESCRIPTION
**Pull request by stdweird**
Fixes (harmless) error message in sssd logs

### CODE REVIEW

File: ncm-authconfig/src/main/pan/components/authconfig/sssd.pan [hunk 1]
```diff
@@ -117,7 +117,7 @@ type authconfig_sssd_domain  = {
     "max_id" : long = 0
     "enumerate" : boolean = false
     "timeout" : long = 10
-    "force_timeout" : long = 60
+    "force_timeout" ? long with {deprecated(0, "Unsupported in later versions"); true}
```
Comment (ned21) [inline ncm-authconfig/src/main/pan/components/authconfig/sssd.pan]: This results in a context-less warning message which does not mention the path. Can you please change it to say something like, “Warning: sssd/force_timeout option is unsupported in later releases of sssd and will be removed from the next release of Quattor”

**Comment by stdweird** [comment]
jenkins test this please

**Comment by jrha** [comment]
@stdweird are you able respond to @Pansanel's comment?

**Comment by jrha** [comment]
Looks like it was removed in https://github.com/SSSD/sssd/commit/5b0735876aa66464b24cb7736a74fafd8ec82128 which seems to correspond to v1.14.2

**Review by ned21 — APPROVED** [review]

**Pull request merged**